src/uct/api/uct.h

```diff
@@ -921,6 +924,14 @@ struct uct_iface_attr {
     uct_linear_growth_t      latency;      /**< Latency model */
     uint8_t                  priority;     /**< Priority of device */
     size_t                   max_num_eps;  /**< Maximum number of endpoints */
+    unsigned                 dev_num_paths;/**< How many different network paths
```

Comment:
Does not have to be different ? For sockets I don't think they use "different". I think it is  an _optimal_ number of network paths that can be used for maximum utilization of the network path ?

Reply:
For more advanced users it might be useful to have a query to translate the index to actually HW resource, like LMC bit. Potentially user may want to configure this things from command line. I think we are missing query interface for this.

Reply:
@shamisp comment is updated, also command line configuration will be available for each transport using env vars (for example today we can we which LMC values to use in https://github.com/openucx/ucx/blob/master/src/uct/ib/base/ib_iface.c#L159

Reply:
But for user , there is no way to translate UCP index to UCT transport Id. I think this might be useful function to have. Otherwise there is no idea how UCP level is gets mapped to UCT level. From configuration perspective, I think this has to be controlled on UCP level ?

Reply:
not sure i understand. The UCP user API is not changed. UCP will create "dev_num_paths" endpoints, with path_index parameter ranging from 0..dev_num_paths-1, and use them for multi-rail logic.

Reply:
My though that we want to have UCP level configuration that will let user to control the lane by setting index 0-n. I guess, what you suggest is not to have the option and just have this control on UCT level , with no direct exposure in UCP ?

Reply:
Yes. I guess we can also add UCP configuration for this if needed, not sure if it's worth the extra complication. In any case, i think it does not affect the UCT API.